packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -474,9 +486,20 @@ const defaultConfigs = {
   XAUUSD: {
     type: "tradermade",
     pair: "XAUUSD",
-    minTimeBetweenUpdates: 60,
+    // Since TradeMadeAPI timeseries data is unavailable over the weekend,
```

Comment:
do we want to repeat this? we could also just put the comment within the feed I guess?

Reply:
Good point, I'll refactor to the feed